pymc3/tests/test_minibatches.py

```diff
@@ -132,10 +132,10 @@ def gen():
 
     def test_pickling(self, datagen):
         gen = generator(datagen)
-        pickle.loads(pickle.dumps(gen))
+        cloudpickle.loads(cloudpickle.dumps(gen))
         bad_gen = generator(integers())
-        with pytest.raises(Exception):
-            pickle.dumps(bad_gen)
+        with pytest.raises(TypeError):
```

Comment:
Maybe add a `match="..."`?

Reply:
I had that actually (in 4740bb21a08e24c6b1275959229ffaf1f4a0908a), but had to take it out because apparently the error message is different on different platforms (specifically: "cannot pickle" versus "can't pickle" 🤦). Didn't want to fail on that, obviously.

Reply:
Ah okay. And according to the CI logs it's _very_ different.
Then let's go without it.